packages/smooth_app/lib/generic_lib/bottom_sheets/smooth_bottom_sheet.dart

```diff
@@ -402,7 +404,7 @@ class SmoothModalSheetHeader extends StatelessWidget implements SizeWidget {
     return IconTheme(
       data: IconThemeData(color: tintColor),
       child: Container(
-        height: suffix is SmoothModalSheetHeaderButton ? double.infinity : null,
+        height: null,
```

Comment:
Have you checked that you don't break other parts of the app?
This is used for help messages in the Edit mode.